aemedge/blocks/tabs/tabs.css

```diff
@@ -271,3 +272,98 @@
   }
 
 }
+
+/* stylelint-disable-next-line no-descending-specificity */
+.tabs.big .tabs-list button p {
+  font-weight: 600;
+}
+
+.tabs.big .accordion-wrapper {
+  max-width: 1130px;
+  margin: 1rem auto;
+  padding: 0 1rem;
+}
+
+.tabs.big .accordion .details .summary {
+  padding: 1.25rem 2rem 1.25rem 0;
+  border-bottom: 1px solid rgb(23 23 37 / 10%);
+  font-size: 1.5rem; 
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.tabs.big .accordion .details .summary::before {
+  content: '';
+  border: solid #454550;
+  border-width: 0 3px 3px 0;
+  height: 10px;
+  width: 10px;
+  pointer-events: none;
+  position: absolute;
+  right: 9px;
+  top: 50%;
+  transform: translateY(-50%) rotate(-135deg);
+  transition: transform 0.3s ease;
+  background: none;
+}
+
+/* Rotate arrow when accordion is open */
+.tabs.big .accordion .details.open .summary::before {
+  transform: translateY(-50%) rotate(45deg);
```

Comment:
Make this rotate(-135deg) and then it will match the position of the mobile footer accordion arrow positions.